pp/event/activate.go

```diff
@@ -39,7 +40,7 @@ func Activate(amount, fee, gas int64) error {
 		}
 	}
 
-	utils.Log("Sending activate message to SP! " + activateReq.PpInfo.P2PAddress)
+	utils.Log("Sending activate message to SP: " + client.SPConn.GetName() + ", from: " + activateReq.PpInfo.P2PAddress)
```

Comment:
Maybe add a nil check here, because the SendMessageToSPServer function has logic to connect to SP if SPConn is nil, so it's probably not impossible for it to still be nil here